### DISCUSSION TITLE
grafana failure

### DISCUSSION THREAD

**Discussion by issmonitor** [Security-Onion-Solutions/securityonion#3273] [Unsupported Versions]
Greetings,

I am having an issue.  Been testing out SO as a distributed setup and have to replace one of the sensors due to not enough horsepower of the sensor we tested.   In the process I used the same name convention when replacing one sensor with another.  In the processes of attaching the new sensor and it seems to have broken grafana and all services are in a "WAIT_START" state.  When I try to start so-grafana I get the following.   Is there a way to remove a unwanted sensor or is this going to be a reinstall of SO?

Thanks 

```
[CRITICAL] Rendering SLS 'base:grafana' failed: while constructing a mapping
  in "<unicode string>", line 17, column 1
found conflicting ID 'dashboard-<servername>'
  in "<unicode string>", line 169, column 1
local:
    Data failed to compile:
----------
    Rendering SLS 'base:grafana' failed: while constructing a mapping
  in "<unicode string>", line 17, column 1
found conflicting ID '<servername>'
  in "<unicode string>", line 169, column 1
```

**Comment by m0duspwnens** [comment]
@issmonitor Can you check on the manager for duplicate entries in `/opt/so/saltstack/local/pillar/data/sensorstab.sls` and remove them if present. Once done, run `sudo salt-call state.apply grafana`.
